File: app/src/main/java/com/wr15/mytestproject/model/MFilm.java
package com.wr15.mytestproject.model;

import com.google.gson.annotations.SerializedName;

public class MFilm {

    @SerializedName("id")
    int id;
    @SerializedName("title")
    private String title;
    @SerializedName("vote_average")
    private double vote_average;
    @SerializedName("overview")
    private String Overview;
    @SerializedName("release_date")
    private String release_date;
    @SerializedName("poster_path")
    private String poster_path;
    @SerializedName("backdrop_path")
    private String backdrop_path;
    @SerializedName("popularity")
    private String popularity;


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public double getVote_average() {
        return vote_average;
    }

    public void setVote_average(double vote_average) {
        this.vote_average = vote_average;
    }

    public String getOverview() {
        return Overview;
    }

    public void setOverview(String overview) {
        Overview = overview;
    }

    public String getRelease_date() {
        return release_date;
    }

    public void setRelease_date(String release_date) {
        this.release_date = release_date;
    }

    public String getPoster_path() {
        return poster_path;
    }

    public void setPoster_path(String poster_path) {
        this.poster_path = poster_path;
    }

    public String getBackdrop_path() {
        return backdrop_path;
    }

    public void setBackdrop_path(String backdrop_path) {
        this.backdrop_path = backdrop_path;
    }

    public String getPopularity() {
        return popularity;
    }

    public void setPopularity(String popularity) {
        this.popularity = popularity;
    }
}

File: app/src/main/java/com/wr15/mytestproject/model/MFilm.kt
package com.wr15.mytestproject.model

import com.google.gson.annotations.SerializedName

class MFilm {
    @SerializedName("id")
    var id = 0

    @SerializedName("title")
    var title: String? = null

    @SerializedName("vote_average")
    var vote_average = 0.0

    @SerializedName("overview")
    var overview: String? = null

    @SerializedName("release_date")
    var release_date: String? = null

    @SerializedName("poster_path")
    var poster_path: String? = null

    @SerializedName("backdrop_path")
    var backdrop_path: String? = null

    @SerializedName("popularity")
    var popularity: String? = null
}